### ISSUE TITLE
make search field more promient

### ISSUE THREAD

**Issue by wilzbach** [dlang/dub-registry#162]
Imho it would increase the UX a lot if the search field would be __a lot__ bigger, centered and part of the content. We all know and love the best solution to this:

![image](https://cloud.githubusercontent.com/assets/4370550/15628361/56b31bba-24ff-11e6-9e6a-23a0bdc8e78a.png)

**Comment by wilzbach** [comment]
Another good example from npm:

![image](https://cloud.githubusercontent.com/assets/4370550/15657541/26bbe9f0-26b2-11e6-82fa-7efb0c00cf40.png)

**Comment by MartinNowak** [comment]
Yes, search needs to be central. Let's move the menu and docs somewhere else first.
Categories (tags) should just be part of search (maybe differently highlighted suggestions).

Recently found https://codepen.io/webmatze/pen/isuHh which looks small enough to do it in plain JS, but some of the shelf select2 et.al. might work as well, if we're willing to accept the cost of jQuery. Those would mainly be interesting to edit categories/tags, but could be useful for search as well.

**Comment by MartinNowak** [comment]
Not to mention auto-focus of the search field :).

**Comment by MartinNowak** [comment]
Should be with typeahead (#161) and tag (#398) support.
